src/map-config.ts

```diff
@@ -0,0 +1,38 @@
+export type MapConfig = {
+  tileUrl: string;
+  attribution: string;
+}
+
+export const DefaultMapConfig: MapConfig = {
+  tileUrl: '//{s}.tile.jawg.io/jawg-terrain/{z}/{x}/{y}.png?access-token=t6fAKnvaPdPCucraY88YwlKjBfUHqBMvvZBIWlcp1Z9Z5FVtA02uWo6Dc9DGB2JO',
+  attribution: 'Map &copy; <a href="http://jawg.io" target="_blank" class="jawg-attrib"><b>Jawg</b>Maps</a> | Map data &copy; <a href="https://www.openstreetmap.org/copyright" target="_blank" class="osm-attrib">OpenStreetMap contributors</a>',
+};
+
+function maybeFixMapConfig(mapConfig: MapConfig) {
+  // default jawg url is specified with "//" which fails when loading built index.html
+  // as a file
+  if (window.location.protocol === 'file:' && !mapConfig.tileUrl.includes('http:')) {
+    return {
+      ...mapConfig,
+      tileUrl: `http:${mapConfig.tileUrl}`,
+    };
+  }
+  return mapConfig;
+}
+
+async function fetchMapConfig(): Promise<MapConfig> {
+  try {
+    const response = await fetch('/frontend/config');
```

Comment:
This can be with a relative path ? It will fail if someone is exposing its pelias API on a non root path
e.g if the base pelias API is `https://api.example.com/geocoding/`, it should call `https://api.example.com/geocoding/frontend/config`

Can we add the extension `config.json`?

Reply:
that makes sense. I was/am struggling with how this is hosted in pelias-api, where the statically generated files get mounted at /frontend, so trying to find a relative path of 'frontend/config' would have it looking at 'frontend/frontend/config' which is not right. Maybe it should just look at './config' ?